### ISSUE TITLE
Named Entity Recognition

### ISSUE THREAD

**Issue by rain1024** [magizbox/underthesea#90]
* Under The Sea version: 1.0.20
* Python version: 2.7
* Operating System:

### Description

Feature Request: Please add named entity recognition function to underthesea

**Comment by baonc** [comment]
Hi, may I implement this issue ?

**Comment by rain1024** [comment]
yes. of course. How is your progress so far?

**Comment by baonc** [comment]
I found your project on reddit this morning. I am very interesting with this and I have just read your code, and see that you used CRFSuite of Chokkan so much [need some reference here (hehe)]. I think I will use CRF too (I have experience with this - almost project I did in my old companies use CRF like [ASPECT, TERM] detection (for sentiment analysis), NER and WordSegmenter) or RNN (new trending - and also very impressive result), but first, I need to labeling data. haha.
